global.json

```
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "10.0.100-preview.5.25277.114",
+    "version": "10.0.100-preview.6.25315.102",
```

```suggestion
    "version": "10.0.100-preview.5.25277.114",
```
Let's keep the released SDK version for now.